### DISCUSSION TITLE
Tailwind 4

### DISCUSSION THREAD

**Discussion by ssijak** [heroui-inc/heroui#4135] [General]
How big of a lift would be to upgrade to tailwind 4 when it releases? Beta is out https://tailwindcss.com/docs/v4-beta#getting-started

**Comment by jrgarciadev** [comment]
Already in our Roadmap 🙌🏼

[a comment by an automated account is not shown]

**Reply by J4v4Scr1pt** [reply]
> @jrgarciadev is this avaialble? Couldn't find info on the [roadmap](https://nextuioss.featurebase.app/roadmap/status)

They have stated this on several occasions but latest is in this [post](https://www.heroui.com/blog/introducing-heroui#whats-next). 👍 
![image](https://github.com/user-attachments/assets/b19f7e43-5d8f-44b7-8223-2205348b1c38)

**Comment by wingkwong** [comment]
PR: https://github.com/heroui-inc/heroui/pull/4656
